<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑 Light</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CppDNCompressor v1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>301</width>
      <height>681</height>
     </rect>
    </property>
    <property name="title">
     <string>源代码</string>
    </property>
    <widget class="QTextEdit" name="srcCode_textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>281</width>
       <height>631</height>
      </rect>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>501</width>
      <height>681</height>
     </rect>
    </property>
    <property name="title">
     <string>目标代码</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>481</width>
       <height>631</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="compressWidget">
      <attribute name="title">
       <string>压缩代码</string>
      </attribute>
      <widget class="QTextEdit" name="compressCode_textEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>461</width>
         <height>531</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_compress">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>550</y>
         <width>381</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>554</y>
         <width>72</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>文件路径：</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="decompressWidget">
      <attribute name="title">
       <string>解压缩代码</string>
      </attribute>
      <widget class="QTextEdit" name="decompressCode_textEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>461</width>
         <height>531</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>554</y>
         <width>72</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>文件路径：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_decompress">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>550</y>
         <width>381</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>应用输出</string>
      </attribute>
      <widget class="QTextEdit" name="console_textEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>461</width>
         <height>571</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>141</width>
      <height>661</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>120</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>文件</string>
     </property>
     <widget class="QPushButton" name="pushButton_openfile">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>40</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_closefile">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>120</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>源代码</string>
     </property>
     <widget class="QPushButton" name="pushButton_compress">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>40</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>压缩</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_decompress">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>解压</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>120</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>目标代码</string>
     </property>
     <widget class="QPushButton" name="pushButton_eSaveTargetCode">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>另存为</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_N">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="action_Openfile"/>
    <addaction name="action_ESavefile"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="action_ClearAll"/>
   </widget>
   <addaction name="menu_N"/>
   <addaction name="menu_E"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Newfile">
   <property name="text">
    <string>新建(N)</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Openfile">
   <property name="text">
    <string>打开(O)</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Savefile">
   <property name="text">
    <string>保存(S)</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_ESavefile">
   <property name="text">
    <string>另存为(E)</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>退出(Q)</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
  </action>
  <action name="action_ClearAll">
   <property name="text">
    <string>清空全部(C)</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+C</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
